src/main/java/de/hysky/skyblocker/config/categories/GeneralCategory.java

```diff
@@ -155,6 +156,14 @@ public static ConfigCategory create(SkyblockerConfig defaults, SkyblockerConfig
                                         newValue -> config.general.itemTooltip.enableBazaarPrice = newValue)
                                 .controller(ConfigUtils::createBooleanController)
                                 .build())
+                        .option(Option.<GeneralConfig.Craft>createBuilder()
+                                .name(Text.translatable("skyblocker.config.general.itemTooltip.craft"))
+                                .binding(defaults.general.itemTooltip.enableCraftingCost,
+                                        () -> config.general.itemTooltip.enableCraftingCost,
+                                        newValue -> config.general.itemTooltip.enableCraftingCost = newValue)
+                                .listener((Option<GeneralConfig.Craft> craft, GeneralConfig.Craft newCraft) -> CraftPriceTooltip.clearCache())
```

Comment:
Could simplify lambda arguments for readability, like so: `(ignored, ignored2) ->`